code/game/jobs/job/support_chaplain.dm

```diff
@@ -64,11 +64,6 @@
 				B.name = "Uplifting Primer"
 			if("toolboxia")
 				B.name = "Toolbox Manifesto Robusto"
-			if("homosexuality")
-				B.name = "Guys Gone Wild"
-			if("lol", "wtf", "gay", "penis", "ass", "poo", "badmin", "shitmin", "deadmin", "cock", "cocks")
-				B.name = pick("Woodys Got Wood: The Aftermath", "War of the Cocks", "Sweet Bro and Hella Jef: Expanded Edition")
-				H.setBrainLoss(99) // starts off retarded as fuck
```

Comment:
In my opinion, the latter one should stay, though the name of the books should probably be changed, along with potentially a few of the descriptors to get it.

Reply:
Making them get braindamage is still a reward for them. I'm fine either way to be honest. Currently out of creativity for names or such so I'm open for suggestions :D